pkg/tools/gen/gendoc.go

```diff
@@ -80,46 +82,47 @@ func (g *GenContext) render(spec *SwaggerV2Spec) error {
 	}
 
 	// collect all the packages and schema list that they contain
-	pkgs := make(map[string]*KclPackage)
-
+	rootPkg := &KclPackage{
+		Name:        spec.Info.Title,
+		Version:     spec.Info.Version,
+		Description: spec.Info.Description,
+	}
 	for _, schema := range spec.Definitions {
 		pkgName := schema.KclExtensions.XKclModelType.Import.Package
-		if _, ok := pkgs[pkgName]; ok {
-			pkgs[pkgName].Schemas = append(pkgs[pkgName].Schemas, schema)
-		} else {
-			pkgs[pkgName] = &KclPackage{
-				Name: pkgName,
+		if pkgName == "" {
+			addOrCreateSchema(rootPkg, schema)
```

Comment:
For line 92-110, maybe we can use a smaller name function called findPackage? Then call `addOrCreateSchema(rootPkg.findPackage(), schema)` or just using `rootPkg.addSchema(schema)`?

Reply:
good suggestion, updated